test/e2e/upstream_serving_operator_test.go

```diff
@@ -0,0 +1,33 @@
+// +build e2e
+
+package e2e
+
+import (
+	"testing"
+
+	"github.com/openshift-knative/serverless-operator/test"
+	upstreame2e "knative.dev/serving-operator/test/e2e"
+	upstreamtest "knative.dev/serving-operator/test"
+)
+
+func TestUpstreamKnativeServingOperator(t *testing.T) {
+	upstreamtest.ServingOperatorNamespace = "knative-serving"
+	suite := upstreame2e.ComplianceSuite()
```

Comment:
This looks elegant but hasn't been proposed upstream. I'm not sure it would even be acceptable. Any reason why we're not just running `go test` on the upstream test and make them configurable to the extent we need?

Reply:
In that way I was able to fine tune what is executed and what isn't - I needed to skip `configure` subtest due to SRVKS-241